src/main/fc/core.h

```diff
@@ -67,6 +67,11 @@ void updateArmingStatus(void);
 
 void taskMainPidLoop(timeUs_t currentTimeUs);
 
+#if defined(USE_RUNAWAY_TAKEOFF) || defined(USE_GPS_RESCUE)
```

Comment:
Declarations in header files are generally not to be made conditional.

Reply:
fixed